src/apps/pillarx-app/index.tsx

```diff
@@ -9,14 +9,19 @@ import { Projection } from '../../types/api';
 // hooks
 import { useWalletAddress } from '@etherspot/transaction-kit';
 import { useGetTilesInfoQuery } from './api/homeFeed';
+import { useGetWaitlistQuery } from '../../services/pillarXApiWaitlist';
+
+// utils
+import { componentMap } from './utils/configComponent';
 
 // components
-import { useGetWaitlistQuery } from '../../services/pillarXApiWaitlist';
-import PillarXLogo from './components/PillarXLogo/PillarXLogo';
 import SkeletonTiles from './components/SkeletonTile/SkeletonTile';
 import H1 from './components/Typography/H1';
+
+
```

Comment:
Just a small thing:

```suggestion

```

...for consistency. I know we need to get Prettier in place, hoping to fit that in this sprint

Reply:
done